scene/main/node.cpp

```diff
@@ -2586,6 +2586,27 @@ void Node::_propagate_reverse_notification(int p_notification) {
 	data.blocked--;
 }
 
+void Node::add_connection_owner(Node *p_owner, Node *p_to_node, const StringName &p_signal_name, const Callable &p_callable, bool is_inherited) {
+	uint32_t hash = HashMapHasherDefault::hash(p_to_node) ^ p_signal_name.hash() ^ p_callable.get_method().hash();
+	if (is_inherited) {
+		// this connection exists from instantiation but is inherited, so return some pointer that
```

Comment:
```suggestion
		// This connection exists from instantiation but is inherited, so return some pointer that
```
Applies to all comments.